runtime/onert/backend/train/ops/PadLayer.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_TRAIN_OPS_PADLAYER_H__
+#define __ONERT_BACKEND_TRAIN_OPS_PADLAYER_H__
+
+#include <backend/IPortableTensor.h>
+#include "OperationUtils.h"
+
+#include <exec/train/ITrainableFunction.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace train
+{
+namespace ops
+{
+
+// Note, this is pad with mode=`CONSTANT`: it doesn't support `REFLECT` and
+// `SYMMETRIC`
+class PadLayer : public ::onert::exec::train::ITrainableFunction
```

Comment:
Q: Is there a reason - this layer does not inherit `cpu::ops::PadLayer`? 
I thought that - Since `forward()` isn't different from `cpu::ops::PadLayer::run()`, we could recduce the code a bit. 

BTW, I'm also happy with the current implementation.

Reply:
That's a great point. Now I figure out!